Not Found: /
Not Found: /
Not Found: /favicon.ico
Not Found: /favicon.ico
Not Found: /api-auth
Not Found: /api-auth
Not Found: /api/v1/registration/
Method Not Allowed: /api/v1/auth/registration/
{'non_field_errors': [ErrorDetail(string='User with email exists', code=403)]}
Bad Request: /api/v1/auth/registration/
Not Found: /api/v1/estaurant/
Unauthorized: /api/v1/restaurant/
Internal Server Error: /api/v1/restaurant/
Traceback (most recent call last):
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 511, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 423, in finalize_response
    assert isinstance(response, HttpResponseBase), (
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'NoneType'>`
Internal Server Error: /api/v1/restaurant/
Traceback (most recent call last):
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 511, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 423, in finalize_response
    assert isinstance(response, HttpResponseBase), (
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'NoneType'>`
Internal Server Error: /api/v1/restaurant/
Traceback (most recent call last):
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 511, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 423, in finalize_response
    assert isinstance(response, HttpResponseBase), (
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'NoneType'>`
Internal Server Error: /api/v1/restaurant/
Traceback (most recent call last):
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 511, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 423, in finalize_response
    assert isinstance(response, HttpResponseBase), (
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'NoneType'>`
Internal Server Error: /api/v1/restaurant/
Traceback (most recent call last):
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 511, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 423, in finalize_response
    assert isinstance(response, HttpResponseBase), (
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'NoneType'>`
Internal Server Error: /api/v1/restaurant/
Traceback (most recent call last):
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 511, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 423, in finalize_response
    assert isinstance(response, HttpResponseBase), (
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'NoneType'>`
Unauthorized: /api/v1/restaurant/
Internal Server Error: /api/v1/restaurant/
Traceback (most recent call last):
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/avee/workspace/dosier/bongo-restaurant/venv/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/avee/workspace/dosier/bongo-restaurant/restaurant/views.py", line 31, in post
    logger.success("resturant created")
AttributeError: 'Logger' object has no attribute 'success'
Bad Request: /api/v1/restaurant/
Bad Request: /api/v1/restaurant/
Bad Request: /api/v1/restaurant/
